misc/scripts/install-local.sh

```diff
@@ -745,8 +795,8 @@ fi
 
 # Running `-B` on a deb package doesn't make sense, so let's download instead
 if ((PACSTALL_INSTALL == 0)) && [[ ${name} == *-deb ]]; then
-    if ! download "${url}"; then
-        fancy_message error "Failed to download '${url}'"
+    if ! download "${source[0]}"; then
+        fancy_message error "Failed to download '${source[0]}'"
```

Comment:
We need to check all the sources, not just the first.

Reply:
debs will always only be singular source, no issue here 👍